SPRCLI/Package.resolved

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Committing `Package.resolved` for the Registry CLI so that developers are running the same package versions as the build pipeline.